features/cellular/TESTS/api/cellular_sms/main.cpp

```diff
@@ -195,21 +196,28 @@ static void test_get_sms()
 
     wait(7);
```

Comment:
I appreciate these tests originated from ARM , but I don't understand why these wait(7) calls are in here.... especially this one which seems to just wait for 7 seconds before it does anything at all. Strange. Do you know the reason for these?

Reply:
It is not explained but I think it is giving time to previous test `test_sms_send` to properly process and complete.